backend/src/database/repositories/memberAttributeSettingsRepository.ts

```diff
@@ -0,0 +1,230 @@
+import lodash from 'lodash'
+import Sequelize from 'sequelize'
+import SequelizeRepository from './sequelizeRepository'
+import { IRepositoryOptions } from './IRepositoryOptions'
+import Error404 from '../../errors/Error404'
+import SequelizeFilterUtils from '../utils/sequelizeFilterUtils'
+
+const Op = Sequelize.Op
+
+class MemberAttributeSettingsRepository {
+  static async findById(id, options: IRepositoryOptions) {
+    const transaction = SequelizeRepository.getTransaction(options)
+
+    const include = []
```

Comment:
I would remove this `include` property since it's not required by `findOne` method in order to make the method simpler and not confuse other coders when they encounter it.

Reply:
redundant includes removed